src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/RectangleSelectionRenderer.js

```diff
@@ -0,0 +1,208 @@
+// @flow
+import React from 'react';
+import type {Normalizer, RectangleChange, SelectionData} from './types';
+import ModifiableRectangle from './ModifiableRectangle';
+import PositionNormalizer from './normalizers/PositionNormalizer';
+import RatioNormalizer from './normalizers/RatioNormalizer';
+import RoundingNormalizer from './normalizers/RoundingNormalizer';
+import SizeNormalizer from './normalizers/SizeNormalizer';
+import withPercentageValues from './withPercentageValues';
+
+type Props = {
+    backdrop: boolean,
+    containerHeight: number,
+    containerWidth: number,
+    disabled: boolean,
+    label?: string,
+    minHeight: number | typeof undefined,
+    minSizeNotification: boolean,
+    minWidth: number | typeof undefined,
+    onChange: (s: ?SelectionData) => void,
+    round: boolean,
+    usePercentageValues: boolean,
+    value: SelectionData | typeof undefined,
+};
+
+class RectangleSelectionRenderer extends React.Component<Props> {
+    static defaultProps = {
+        backdrop: true,
+        disabled: false,
+        minHeight: undefined,
+        minSizeNotification: true,
+        minWidth: undefined,
+        round: true,
+        usePercentageValues: false,
+    };
+
+    get value() {
```

Comment:
That could probably be also a computed getter.

Reply:
fixed